DenunciaCiudadana/src/actions/complaintsAction.js

```diff
@@ -0,0 +1,24 @@
+import {types} from '../types/type';
+import firestore from '@react-native-firebase/firestore';
+
+export const startLoadingComplaints = () => {
+  return async dispatch => {
+    const getComplaints = await firestore().collection('complaints').get();
+    const complaints = await [];
+
+    getComplaints.forEach(doc => {
+      complaints.push({
+        id: doc.id,
+        ...doc.data(),
+      });
+    });
```

Comment:
Why `forEach`? Did you try with `.map`?

Reply:
It is the function that appear in the documentation in firebase.com, for make that  kind of operation with data